src/data/blinkComponents.json

```diff
@@ -0,0 +1,1277 @@
+{
+  "AAA-README": [
+    "", 
+    "This file is generated by src/tools/checkteamtags/extract_components.py", 
```

Comment:
I would pull from https://blinkcomponents-399b3.firebaseapp.com/blinkcomponents for now. You can format it using https://jsonformatter.curiousconcept.com/